### DISCUSSION TITLE
Help...... filament runout sensor is working, but a I cannot resume

### DISCUSSION THREAD

**Discussion by chuckkay** [OpenNeptune3D/OpenNept4une#131] [Q&A]
sensor works, print plays tune and parks, i replace the filament, insert through sensor and into toolhead, sensor registers as filament detected, then i hit resume on screen or on webui and nothing happens, and the tempreture of the bed and nozzle drop to zero...   also console displays: 
10:20 PM
idle_timeout: Timeout set to 600.00 s
10:18 PM
idle_timeout: Timeout set to 7200.00 s

**Comment by halfmanbear** [comment]
Yes the filament sensor works in this case (I'm aware of the pause resume issues) but I also need to disable the filament sensor at initial startup (& in print end / cancel). Also enable sensor in print start

**Comment by halfmanbear** [comment]
https://github.com/OpenNeptune3D/OpenNept4une/issues/133#issue-2160853033

**Comment by halfmanbear** [comment]
https://github.com/OpenNeptune3D/OpenNept4une/pull/134

**Reply by chuckkay** [reply]
Works great now thanks
